jest.setup.ts

```diff
@@ -1 +1,5 @@
 import '@testing-library/jest-dom/extend-expect'
+import dotenv from 'dotenv';
+dotenv.config({
+  path: '.env.test',
```

Comment:
So this works now? Like it automatically uses `.env.test` when testing?

Reply:
That's right!